hooks/post/30-upgrade.rb

```diff
@@ -38,6 +38,15 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
+def remove_elasticsearch
+  gems = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar', 'lucene3']
+  gems.each do |gem|
+    Kafo::Helpers.execute('rpm -e #{gem}')
+  end
+  echo "Elasticsearch has been removed as a dependency, the database files can be removed manually."
+  echo "Remove elastic search files with #rm -rf /var/lib/elasticsearch"
```

Comment:
Works for me!

On Mon, Nov 23, 2015 at 11:02 AM, Eric D Helms <notifications@github.com>
wrote:

> In hooks/post/30-upgrade.rb
> <https://github.com/Katello/katello-installer/pull/267#discussion_r45620571>
> :
>
> > @@ -38,6 +38,15 @@ def import_subscriptions
> >    Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
> >  end
> >
> > +def remove_elasticsearch
> > +  gems = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar', 'lucene3']
> > +  gems.each do |gem|
> > +    Kafo::Helpers.execute('rpm -e #{gem}')
> > +  end
> > +  echo "Elasticsearch has been removed as a dependency, the database files can be removed manually."
> > +  echo "Remove elastic search files with #rm -rf /var/lib/elasticsearch"
>
> We could check for it's existence and only put out the message if found?
> and/or if RPMs were removed?
>
> Also, can't use echo here -- that's not a function in Ruby, better to use:
>
>    Kafo::Helpers.log_and_say :info, message
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello-installer/pull/267/files#r45620571>.
>